author.php

```diff
@@ -34,14 +34,7 @@
         'width' => '300',
         'height' => '300',
     ];
-
```

Comment:
Noticing that this value is not being used by us, only by a couple of NROs. Do we really need to add it? What if we ask them to include into their themes instead?
https://github.com/search?q=org%3Agreenpeace%20author_share_buttons&type=code

Reply:
I think @comzeradd would know better about this, especially if we need to warn the NROs about the change!